# Arquivo: curso-programacao-python\07-tratamento-excecoes-python\try_except_finally_raise.py

print("Arquivo try_except_finally_raise.py criado.")
